debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/antlr/listener/ColumnDefinitionParserListener.java

```diff
@@ -62,33 +64,44 @@ public Column getColumn() {
 
     @Override
     public void enterColumnDefinition(MySqlParser.ColumnDefinitionContext ctx) {
+        uniqueColumn = false;
+        optionalColumn = null;
         resolveColumnDataType(ctx.dataType());
         super.enterColumnDefinition(ctx);
     }
 
     @Override
-    public void enterUniqueKeyColumnConstraint(MySqlParser.UniqueKeyColumnConstraintContext ctx) {
-        if (!tableEditor.hasPrimaryKey()) {
+    public void exitColumnDefinition(MySqlParser.ColumnDefinitionContext ctx) {
```

Comment:
Don't we have to reset the `uniqueColumn` and `optionalColumn` when exiting the column definition?

Reply:
It is reset in `enterColumnDefinition`

Reply:
Ah yes, thanks!